### PR TITLE
Happy path end to end test

### PR DESCRIPTION
**Pull request by jeremyauclert**
## JIRA
[MOB-2154](https://karhoo.atlassian.net/browse/MOB-2154)

## DESCRIPTION
Happy path end to end test 
## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/androidTest/java/com/karhoo/uisdk/address/AddressRobot.kt [hunk 1]
```diff
@@ -24,12 +24,6 @@ class AddressRobot : BaseTestRobot() {
                 text = searchTerm)
     }
 
-    fun selectFirstAddress() {
```
Comment (EdwardW2) [inline uisdk/src/androidTest/java/com/karhoo/uisdk/address/AddressRobot.kt:27]: Was this meant to be removed? Is it not used anywhere?

File: app/src/main/java/com/karhoo/karhootraveller/presentation/karhooLabs/KarhooLabsActivity.kt [hunk 2]
```diff
@@ -18,7 +18,6 @@ class KarhooLabsActivity : AppCompatActivity() {
         toolbar.setNavigationOnClickListener { finish() }
         featureFlag = FeatureFlag(this)
         initialiseFeatureFlags()
-        guestCheckoutSwitch.setOnCheckedChangeListener { buttonView, isChecked -> guestCheckoutClicked()}
```
Comment (EdwardW2) [inline app/src/main/java/com/karhoo/karhootraveller/presentation/karhooLabs/KarhooLabsActivity.kt:21]: Same as previous comment, was this meant to be removed?

Comment (jeremyauclert) [reply app/src/main/java/com/karhoo/karhootraveller/presentation/karhooLabs/KarhooLabsActivity.kt:21]: Well spotted. Adding it back in.

File: uisdk/src/androidTest/java/com/karhoo/uisdk/address/AddressRobot.kt (hunk 1 above)
Comment (jeremyauclert) [reply uisdk/src/androidTest/java/com/karhoo/uisdk/address/AddressRobot.kt:27]: It wasn't used.

**Review by EdwardW2 — APPROVED** [review]

**Pull request merged**